s3-vfs/src/main/java/org/pentaho/amazon/s3/provider/S3Provider.java

```diff
@@ -2,7 +2,7 @@
  *
  * Pentaho Big Data
  *
- * Copyright (C) 2002-2023 by Hitachi Vantara : http://www.pentaho.com
+ * Copyright (C) 2002-2022 by Hitachi Vantara : http://www.pentaho.com
```

Comment:
I don't think we would decrement the copyright year unless this is a reverse commit restoring a previous state?

Reply:
fixed